BD-3330: Add FAQ for field-level encryption (#7845)
braze-inc/braze-docs: _docs/_hidden/private_betas/field_level_encryption.md

### What email address should I expect to see in message archiving?

The plaintext email address is included in messaging archiving. These are sent directly to the customer’s cloud storage provider and there may be other personal data included in the email bodies.

### Can I use mail-to list-unsubscribe for subscription management with field-level encryption?

> No. Instead, we recommend using one-click and list-unsubscribe URL to process the unsubscribes from users.

Author: ```suggestion
No. Mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```

> No. Mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.

Author: ```suggestion
No. Using mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```

> No. Using mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.

Reviewer: ```suggestion
No. Using mail-to list-unsubscribe will send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```
